src/plugins/console/public/services/settings.ts

```diff
@@ -30,72 +31,96 @@ export interface DevToolsSettings {
   polling: boolean;
   pollInterval: number;
   tripleQuotes: boolean;
-  historyDisabled: boolean;
+  isHistoryDisabled: boolean;
+  isKeyboardShortcutsDisabled: boolean;
+}
+
+enum SettingKeys {
+  FONT_SIZE = 'font_size',
+  WRAP_MODE = 'wrap_mode',
+  TRIPLE_QUOTES = 'triple_quotes',
+  AUTOCOMPLETE_SETTINGS = 'autocomplete_settings',
+  CONSOLE_POLLING = 'console_polling',
+  POLL_INTERVAL = 'poll_interval',
+  IS_HISTORY_DISABLED = 'is_history_disabled',
+  IS_KEYBOARD_SHORTCUTS_DISABLED = 'is_keyboard_shortcuts_disabled',
 }
 
 export class Settings {
   constructor(private readonly storage: Storage) {}
 
   getFontSize() {
-    return this.storage.get('font_size', DEFAULT_SETTINGS.fontSize);
+    return this.storage.get(SettingKeys.FONT_SIZE, DEFAULT_SETTINGS.fontSize);
   }
 
   setFontSize(size: number) {
-    this.storage.set('font_size', size);
+    this.storage.set(SettingKeys.FONT_SIZE, size);
     return true;
```

Comment:
not related to your PR, but why all setters return `true` ?

Reply:
As far as I know it may throw an exception if the storage is full